react-components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cognite/reveal-react-components",
-  "version": "0.24.3",
+  "version": "0.24.4",
```

Comment:
We bump version in separate PR

Reply:
👍 , reverted in [cf11e43](https://github.com/cognitedata/reveal/pull/4051/commits/cf11e4376decfc0ea521d43940cb1cbc8235d8cc)